sdk/react/ooyalaSkinBridgeListener.js

```diff
@@ -76,7 +76,7 @@ OoyalaSkinBridgeListener.prototype.onSeekStarted = function(e) {
 OoyalaSkinBridgeListener.prototype.onSeekComplete = function(e) {
   Log.log( "onSeekComplete");
   this.skin.setState({
-    playhead: e.playhead,
```

Comment:
A question here. Do we have a plan about doing something with the playhead event? From this PR, I understand that the playhead passed to React is the seekStart position. Is it ok if we just discard it?